test/error.jl

```diff
@@ -93,7 +93,8 @@ end
 @testset "MethodError for methods without line numbers" begin
     try
         eval(Expr(:function, :(f44319()), 0))
-        f44319(1)
+        @Core.latestworld
+        f44319()
```

Comment:
Question: style-wise, should we usually recommend invokelatest or Core.latestworld?
```suggestion
        @invokelatest f44319()
```